capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -178,6 +213,9 @@ public Object getRequestResult(HttpServletRequest request)
         String searchWord = request.getParameter("stringToSearchBy");
         return runSearchStudySetSqlQuery(pool, searchWord);
       }
+      else if(pathInfo.startsWith("/user")) {
```

Comment:
using if here is enough since you would have returned earlier.

Reply:
Done.